admin/tags.md

```diff
@@ -10,30 +10,30 @@ To assign a tag to a product, go to the product detail page by clicking on the p
 
 ![](/assets/admin-products-tags.png "Reaction Commerce tags and navigation")
 
-To create a new tag, click on the “Add Tags” box, type in your tag, and hit enter to save. If the item is a t-shirt, you might want to tag it with relevant keywords, like short sleeve, cotton, crewneck, etc. 
+To create a new tag, click on the “Add Tags” box, type in your tag, and hit enter to save. If the item is a t-shirt, for instance, you might want to tag it with relevant keywords, like short sleeve, cotton, crewneck, etc.
 
-To delete a tag, click the gray “x” icon (![](/assets/guide-icon-deletetag.png "Reaction Commerce delete tag").)
+To delete a tag, click on the gray “x” icon (![](/assets/guide-icon-deletetag.png "Reaction Commerce delete tag")).
 
 To rearrange the order of a tag, click on the three horizontal lines icon (![](/assets/guide-icon-movetag.png "Reaction Commerce move tags")), and simply drag and drop.
 
 ### Hashtag URLs
 
-The flag icon in each tag creates a product link based on that tag. Clicking on the flag generates a URL for your product containing just the words from that tag.
+To create a product link, click on the flag icon for each tag. It'll generates a URL for your product relevant to that tag.
```

Comment:
'generates' should be 'generate'. Won't let me do a quick update without forking, pulling, PRing, sorry.